src/mbed_tools/cli/build.py

```diff
@@ -29,7 +29,21 @@
     default=os.getcwd(),
     help="Path to local Mbed program. By default it is the current working directory.",
 )
-def build(program_path: str, build_type: str, toolchain: str = "", mbed_target: str = "", clean: bool = False) -> None:
+@click.option(
+    "-f", "--flash", is_flag=True, default=False, help="Flash the binary onto a device",
+)
+@click.option(
+    "--hex-file", is_flag=True, default=False, help="Use hex file, this option should be used with '-f/--flash' option",
+)
+def build(
+    program_path: str,
```

Comment:
Should this be a `pathlib.Path`?

Reply:
The default value of `program_path` is `default=os.getcwd()` which returns `str`. So we should keep `program_path` as `str`.